src/components/FormLabel/FormLabel.jsx

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import './FormLabel.scss';
 
 const FormLabel = ({
+  error,
   inputId,
   isFieldRequired,
   labelText,
 }) => {
+  const labelClasses = classNames(
+    error ? 'error' : null,
```

Comment:
Think this can be written as: `'error': error`.

Reply:
Regarding naming conventions: 

Instead of having 'fieldRequired', we do 'isFieldRequired'. So, would we have 'hasError' as opposed to just 'error'?

Reply:
@JCarmine @juanfabrega I left off the prefix for `error` because it's not just a boolean. It also accepts a `string` that when defined will make the input appear red.